index.bs

```diff
@@ -1873,15 +1873,16 @@ given:</p>
 </ol>
 
 <p class="note">A <a>WebVTT line cue setting</a> configures the offset of the cue box from the video
-viewport's edge in the direction opposite to the <a lt="WebVTT cue writing direction">writing
-direction</a>. For horizontal cues, this is the vertical offset from the top of the video viewport.
-The offset is for the <a lt="WebVTT cue line start alignment">start</a>, <a lt="WebVTT cue line
-center alignment">center</a>, or <a lt="WebVTT cue line end alignment">end</a> of the cue box,
-depending on the <a>WebVTT cue line alignment</a> value - <a lt="WebVTT cue line start
-alignment">start</a> by default. The offset can be given either as a percentage of the video
-dimension or as a line number. Line numbers are based on the size of the first line of the cue.
-Positive line numbers count from the start of the video viewport (the first line is numbered 0),
-negative line numbers from the end of the viewport (the last line is numbered &#x2212;1).</p>
+viewport's edge in the direction orthogonal to the <a lt="WebVTT cue writing direction">writing
+direction</a>. For horizontal cues, this is the vertical offset from the top of the video viewport,
+for vertical cues, it's the horizontal offset. The offset is for the <a lt="WebVTT cue line start
+alignment">start</a>, <a lt="WebVTT cue line center alignment">center</a>, or <a lt="WebVTT cue line
+end alignment">end</a> of the cue box, depending on the <a>WebVTT cue line alignment</a> value - <a
+lt="WebVTT cue line start alignment">start</a> by default. The offset can be given either as a
+percentage of the video dimension or as a line number. Line numbers are based on the size of the
+first line of the cue. Positive line numbers count from the start of the video viewport (the first
+line is numbered 0), negative line numbers from the end of the viewport (the last line is numbered
```

Comment:
Is is true that start == (top or left) and end == (bottom or right)? Or are those terms writing mode dependent, i.e. end could be the left in rtl?

Reply:
I'll quote from the spec, which answers your question:

> A line alignment
> An alignment for the cue box’s line, one of:

> Start alignment
> The cue box’s top side (for horizontal cues), left side (for vertical growing right), or right side (for vertical growing left) is aligned at the line.

> Center alignment
> The cue box is centered at the line.

> End alignment
> The cue box’s bottom side (for horizontal cues), right side (for vertical growing right), or left side (for vertical growing left) is aligned at the line.

>By default, the line alignment is set to start.

The line alignment is separate from the text alignment — right-to-left vs. left-to-right cue text does not affect the line alignment.

Reply:
Yes, writing mode dependent as is written in the spec.

Reply:
Thanks, I thought so. I'm not sure why I was asking now, probably I was worried about the meaning of "start" being clear, but it is well defined. This looks fine to me now.